Scalar.FunctionalTests/Tools/ScalarProcess.cs

```diff
@@ -161,6 +161,7 @@ public void DeleteConfig(string key)
                 }
 
                 string result = process.StandardOutput.ReadToEnd();
+                result += process.StandardError.ReadToEnd();
```

Comment:
I believe that right now, we show the `stderr` in the logs, and it is pretty helpful in diagnosing problems.

Should we still include it in the logs? I think so, i.e. something like:

```c#
string stderr = process.StandardError.ReadToEnd();
Console.Error.Write(stderr);
result += stderr
```

Also, I wonder whether we want to scope this `stderr` handling to more narrowly e.g. by introducing a new parameter in the `CallScalar()` function (something like `bool includeStderrInResult = false`) and passing `includeStderrInResult: true` specifically in [the `Diagnose()`](https://github.com/microsoft/scalar/blob/cb07c5e24538c31b365e69e5a70dac4e01a4a7d0/Scalar.FunctionalTests/Tools/ScalarProcess.cs#L80) function.

Reply:
You'll need `processInfo.RedirectStandardError = true;` (there is a similar one for standard out above).

Reply:
> I wonder whether we want to scope this `stderr` handling more narrowly e.g. by introducing a new parameter in the `CallScalar()` function (something like `bool includeStderrInResult = false`) and passing `includeStderrInResult: true` specifically in [the `Diagnose()`](https://github.com/microsoft/scalar/blob/cb07c5e24538c31b365e69e5a70dac4e01a4a7d0/Scalar.FunctionalTests/Tools/ScalarProcess.cs#L80) function.

That would probably help the current failures (see e.g. [this one](https://github.com/microsoft/scalar/runs/7012517724?check_suite_focus=true#step:15:2697)) where the Functional Tests expect an empty output, but there is this warning on `stderr`...